portfolio/src/main/webapp/scripts/results.js

```diff
@@ -1,20 +1,37 @@
 const baseUri = 'https://api.spotify.com/v1';
 const accessTokenKey = 'access_token';
 
-// Hardcoded results.
 var songNum = 1;
-var seedArtists = '7Ln80lUS6He07XvHI8qqHH'; 
-var seedGenres = 'alternative';
-var seedTracks = '5ruzrDWcT0vuJIOMW7gMnW';
 
 // Gets result based on quiz results.
 function getResult() {
-    // Need to change options here to match new quiz and get values from URL query params.
+    // Get values from URL query params.
+    let seedGenres = null;
```

Comment:
One thing to consider - because we set these to null initially and then don't require they be updated before making the spotify api call, if I just directly load the /results.html page without any query parameters it seems to break because the spotify api returns an error.

Related, if I mess with the url params and set the genre param to "blahblah" or some other invalid value, the spotify call succeeds but returns 0 results so then when we try to render the info for the first result, we actually see a console error.

Anyway, updating this stuff can definitely be a separate pull request, but just wanted to mention it while I was thinking about it.